frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/secret.tsx

```diff
@@ -21,6 +21,7 @@ export const GetSecret: React.FC<GetSecretProps> = ({ obj }) => {
     const configMap = k8sGet(ConfigMapModel, name, namespace);
     Promise.all([secret, configMap])
       .then((data) => {
+        const bucketName = _.get(data[1], 'data.BUCKET_NAME');
```

Comment:
use optional chaining.

Reply:
Need to backport to 4.3

Reply:
Will that make sense to do it for a single z release?

Reply:
Let's keep it ready for any case.